api/specs/webserver/openapi-catalog.yaml

```diff
@@ -85,14 +85,14 @@ paths:
         schema:
           title: Service key
           type: string
-          pattern: ^(simcore)/(services)/(comp|dynamic)(/[^\\s/]+)+$
+          pattern: ^(simcore)/(services)/(comp|dynamic|frontend)(/[\w/-]+)+$
       - in: path
         name: service_version
         required: true
         schema:
           title: Service version
           type: string
-          pattern: ^(0|[1-9]\\d*)(\\.(0|[1-9]\\d*)){2}(-(0|[1-9]\\d*|\\d*[-a-zA-Z][-\\da-zA-Z]*)(\\.(0|[1-9]\\d*|\\d*[-a-zA-Z][-\\da-zA-Z]*))*)?(\\+[-\\da-zA-Z]+(\\.[-\\da-zA-Z-]+)*)?$
+          pattern: ^(0|[1-9]\d*)(\.(0|[1-9]\d*)){2}(-(0|[1-9]\d*|\d*[-a-zA-Z][-\da-zA-Z]*)(\.(0|[1-9]\d*|\d*[-a-zA-Z][-\da-zA-Z]*))*)?(\+[-\da-zA-Z]+(\.[-\da-zA-Z-]+)*)?$
```

Comment:
sorry for being suspicious. but is this correct? just that I noticed some weird inconcistencies in my own PR suddenly.. just wondering if you know the cause?

Reply:
This is all auto-generated with the script in ``services/web/server/tests/sandbox/*_openapi_generator.py``
I started doing this to reliably build the webserver API